README.md

```diff
@@ -74,13 +71,265 @@ storyblokInit({
 });
```

Comment:
Is this only true for the US or also for China? If it's also true for China, let's be more generic to include all options here e.g. "if you are using a space located in a region other than the EU (default) use apiOptions.region to specify e.g. "us" or "cn"

Reply:
i have to investigate more about the China configuration. So for now, let me add only the US option. Then i will open a new PR when EU vs US vs CN will be more clear to me